tests/core/gui/test_revoked_user.py

```diff
@@ -0,0 +1,64 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pendulum
+import pytest
+from PyQt5 import QtCore
+from PyQt5.QtWidgets import QLabel
+
+from parsec.api.data import RevokedUserCertificateContent
+from parsec.core.local_device import save_device_with_password
+
+
+@pytest.fixture
+async def logged_gui(
+    aqtbot, gui_factory, running_backend, autoclose_dialog, core_config, bob, alice_user_fs
+):
+    save_device_with_password(core_config.config_dir, bob, "P@ssw0rd")
+    gui = await gui_factory()
+    lw = gui.test_get_login_widget()
+    tabw = gui.test_get_tab()
+
+    assert lw.combo_username.currentText() == f"{bob.organization_id}:{bob.device_id}"
+    await aqtbot.key_clicks(lw.line_edit_password, "P@ssw0rd")
+
+    async with aqtbot.wait_signals([lw.login_with_password_clicked, tabw.logged_in]):
+        await aqtbot.mouse_click(lw.button_login, QtCore.Qt.LeftButton)
+
+    # Users and devices are not populated on startup
+    assert gui.test_get_devices_widget().devices == []
+    assert not gui.test_get_devices_widget().initialized
+    assert gui.test_get_users_widget().users == []
+    assert not gui.test_get_users_widget().initialized
+
+    return gui
+
+
+@pytest.mark.gui
+@pytest.mark.trio
+async def test_revoked_notification(
+    aqtbot, running_backend, autoclose_dialog, logged_gui, alice_user_fs, bob
+):
+    now = pendulum.now()
+    revoked_device_certificate = RevokedUserCertificateContent(
+        author=alice_user_fs.device.device_id, timestamp=now, user_id=bob.user_id
+    ).dump_and_sign(alice_user_fs.device.signing_key)
+    central_widget = logged_gui.test_get_central_widget()
+    assert central_widget is not None
+    async with aqtbot.wait_signals([central_widget.new_notification]):
```

Comment:
Should be `wait_signal` since there's only one.

Reply:
fixed